### PR TITLE
Add links. More on distance metrics without index

### PR DESCRIPTION
**Pull request by kkraune**
CC @jobergum

### CODE REVIEW

**Review by jobergum — APPROVED** [review]

**Comment by jobergum** [comment]
This is pure quality!

File: en/nearest-neighbor-search.md [hunk 1]
```diff
@@ -198,8 +204,16 @@ Vespa supports six different [distance-metrics](reference/schema-reference.html#
...
 
-### Configure rank profiles for nearest neighbor search
+The distance-metric is a property of the field.
+This is obvious when `index` is set on the vector field,
+as the distance metric is used when building the index structure.
+Without `index`, no extra data structures are built for the field,
+and the distance-metric setting is used when calculating the distance at query-time.
+It still makes sense to have the metric as a field-property,
+as the field values are often produced using a specific distance metric.
+Finally, without `index` there is no need to restart Vespa after redeploying with a new distance metric.
```
Comment (kkraune) [inline en/nearest-neighbor-search.md:214]: I am not sure if this is true, @geirst (the restart required, if there is no HNSW index)

Comment (kkraune) [reply en/nearest-neighbor-search.md:214]: Actually, https://docs.vespa.ai/en/reference/schema-reference.html#changes-that-require-restart-but-not-re-feed says a restart is required regardless of an index, will remove the last sentence.

Comment (geirst) [reply en/nearest-neighbor-search.md]: Yes, changing the `distance-metric` triggers a restart action from the config model.

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**